tests/input/test_input_gg_checks.py

```diff
@@ -0,0 +1,81 @@
+"""Test suite for parse_config function."""
+
+import pytest
+from gatorgrade.input.in_file_path import parse_config
+
+
+def test_parse_config_gg_check_in_file_context_contains_file():
+    """Test to make sure gatorgrader checks that have a file context include the file name in the list of lists."""
+    # Given the path to the test yml file
+    config = "tests/input/yml_test_files/gatorgrade_one_gg_check_in_file_context.yml"
+    # when the parse_config function is run
+    output = parse_config(config)
+    # then assert that the file path exists in the first index of the list
+    assert "file.py" in output["gatorgrader"]
+
+
+def test_parse_config_check_gg_matchfilefragment():
+    """Test to make sure keywords like MatchFileFragment appear inside the gator grader check list."""
+    # Given the path to the test yml file
+    config = "test/input/yml_test_files/gatorgrade_matchfilefragment.yml"  # Make fixture that will create yml file to be parsed instead of expected file
+    # When the parse_config is run
+    output = parse_config(config)
+    # Then assert that match file fragment exists
+    assert set(
+        [["--description", "Complete all TODOs"], ["check", "MatchFileFragment"]]
+    ) == set(
+        output["gatorgrader"]
+    )  # Assert that output["gatorgrader"] is equal to expected output
+
+
+def test_parse_config_gg_check_no_file_context_contains_no_file():
+    """Test to make sure gator grader checks that have no file context do not show the file name in the outputted list."""
+    # Given the path to the test yml file
+    config = "tests/input/yml_test_files/gatorgrade_one_gg_check_no_file_context.yml"
+    # When parse_config is run
+    output = parse_config(config)
+
+    # Then assert that there is no file path indicated in the first index of list
+    assert output == ["--description", "Have 8 commits", "CountCommits", "--count", "8"]
```

Comment:
Since `output` will be a dictionary, this assertion should be made on the first index of the `"gatorgrader"` key--`output["gatorgrader"][0]`.

Reply:
yup I will add this!